core/internal/cltest/factories.go

```diff
@@ -520,16 +520,12 @@ func MustInsertRandomKey(
 	keystore.XXXTestingOnlyAdd(key)
 
 	for _, cid := range chainIDs {
-		var nonce int64
 		enabled := true
 		for _, opt := range opts {
 			switch v := opt.(type) {
 			case int:
-				nonce = int64(v)
```

Comment:
Should these cases be removed so we hit default instead?

Reply:
I left them in to avoid the log in the default case since they're known types just not being used. But I agree cleaning up this code is probably better than trying to dodge the log

Reply:
I've removed the extra cases here